src/pages/MyLibrary/LibraryBooks/OpenBook/OpenBook.js

```diff
@@ -0,0 +1,118 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { Page, Document } from 'react-pdf';
+import StarRating from './StarRating';
+import useViewPort from './ViewPort/useViewPort';
+import { pdfjs } from 'react-pdf';
+pdfjs.GlobalWorkerOptions.workerSrc = `//cdnjs.cloudflare.com/ajax/libs/pdf.js/${pdfjs.version}/pdf.worker.js`;
+
+const OpenBook = props => {
+  const { book, setBook } = props;
+  const [numPages, setNumPages] = useState(null);
+  const [pageNumber, setPageNumber] = useState(1);
+  const { width, height } = useViewPort();
+
+  function onDocumentLoadSuccess({ numPages }) {
+    setNumPages(numPages);
+  }
+
+  const closeBook = () => {
+    setBook(book => !book);
+  };
+
+  const bookscale = () => {
+    let newwidth = width / 1920;
+    let newheight = height / 1019;
+
+    return newwidth >= newheight ? newheight : newwidth;
+  };
+
+  return (
+    <>
+      <Modal book={book} onClick={closeBook} />
+      <OpenBookTotal book={book}>
+        <DocumentBox>
```

Comment:
```suggestion
  return (
    book (
      <>
        <Modal onClick={closeBook} />
        <OpenBookTotal>
          <DocumentBox>
```
book 이라는 상태값을 각각의 styled-components로 보내서 관리하기 보다는 조건부 렌더링을 통해 한 번에 관리할 수 있겠네요.

Reply:
말씀주신 대로 다른 코드에서도 display none을 사용하지않고 조건부 렌더링으로 변경했습니다.